modules/pixfutureBidAdapter.js

```diff
@@ -0,0 +1,323 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { getStorageManager } from '../src/storageManager.js';
+import { BANNER } from '../src/mediaTypes.js';
+import { config } from '../src/config.js';
+import includes from 'core-js-pure/features/array/includes.js';
+import * as utils from '../src/utils.js';
+import { auctionManager } from '../src/auctionManager.js';
+import find from 'core-js-pure/features/array/find.js';
+
+const SOURCE = 'pbjs';
+const storageManager = getStorageManager();
+const USER_PARAMS = ['age', 'externalUid', 'segments', 'gender', 'dnt', 'language'];
+export const spec = {
+  code: 'pixfuture',
+  hostname: 'https://prebid-js.pixfuture.com',
+
+  getHostname() {
+    let ret = this.hostname;
+    try {
+      ret = storageManager.getDataFromLocalStorage('ov_pixbidder_host') || ret;
+    } catch (e) {
+    }
+    return ret;
+  },
+
+  isBidRequestValid(bid) {
+    return !!(bid.sizes && bid.bidId && bid.params &&
+                (bid.params.pix_id && (typeof bid.params.pix_id === 'string')));
```

Comment:
This is where you're currently enforcing string

Reply:
Its should be string.

Reply:
Right, so make sure that your [dev-docs](https://github.com/PixFuture-Media-Inc/prebid.github.io/tree/master/dev-docs/bidders/pixfuture.md) ask for a string. Once you update that, Pull Request it too the prebid.github.io repo here https://github.com/prebid/prebid.github.io/pulls